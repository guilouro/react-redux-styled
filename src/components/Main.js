import React from 'react';
import { Container, Title } from '../styles/Main.style';
import reactLogo from '../../images/react.png';

const Main = () => (
    <Container>
        <img src={reactLogo} alt="React" />
        <Title>
            React
            <span>App</span>
        </Title>
        <i>is running...</i>
    </Container>
);

export default Main;
