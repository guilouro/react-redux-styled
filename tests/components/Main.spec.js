import React from 'react';
import { shallow } from 'enzyme';
import Main from '../../src/components/Main';

describe('Should render correctly', () => {
    it('Should render correctly', () => {
        const wrapper = shallow(<Main />);
        expect(wrapper).toMatchSnapshot();
    });
});
