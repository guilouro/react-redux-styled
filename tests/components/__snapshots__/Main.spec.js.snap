// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> Should render correctly 1`] = `
<styled.section>
  <img
    alt="React"
    src={Object {}}
  />
  <styled.div>
    React
    <span>
      App
    </span>
  </styled.div>
  <i>
    is running...
  </i>
</styled.section>
`;
